eknrenderer/eknr-renderer.c

```diff
@@ -769,5 +769,10 @@ eknr_renderer_init (EknrRenderer *self)
 EknrRenderer *
 eknr_renderer_new (void)
 {
+  /* Before creating the renderer, make sure to call bindtextdomain ()
+   * and bind_textdomain_codeset () */
+  bindtextdomain (GETTEXT_PACKAGE, DATADIR "/locale");
```

Comment:
I would name it LOCALEDIR and then do in the meson file:

`conf.set_quoted('PACKAGE_LOCALE_DIR', join_paths(get_option('prefix'), get_option('datadir'), 'locale'))`